interview-questions.md

```diff
@@ -8,50 +8,50 @@ Then, research the question to expand on your answer. Even if you feel you have
 
```

Comment:
Thanks for sharing your responses to the interview questions. Let's find a way to share more content so the interviewer can understand you will be a great asset to the team.
Try this technique:
a) Define the topic
b) Explain how you used it
c) Research something interesting about the topic (if you didn't define it correctly, explain your error)